builtin/providers/aws/resource_aws_elasticache_cluster.go

```diff
@@ -187,6 +205,16 @@ func resourceAwsElasticacheClusterCreate(d *schema.ResourceData, meta interface{
 		req.CacheParameterGroupName = aws.String(v.(string))
 	}
 
+	if !strings.Contains(d.Get("node_type").(string), "cache.t2") {
```

Comment:
I don't believe this is the approach we should take. We should document that snapshotting does not work with `cache.t2`'s , and we should bubble up the error if a user uses t2's anyway. 

Only applying these params in this block would (especially without any message) would be misleading the user, as without an error, anything in their config that doesn't throw an error in applying will be written to state, so from Terraforms view, they are applied. But in reality, they are not.